metadata.platform.in

```diff
@@ -23,6 +23,13 @@ version=@@FDO_RUNTIME_VERSION@@
 directory=lib/extensions/gstreamer-1.0
 subdirectories=true
 
+[Extension org.freedesktop.Platform.Icontheme]
+directory=share/runtime/share/icons
+subdirectories=true
+no-autodownload=true
```

Comment:
I'm not entirely sure we don't want to download this; on one hand, it won't do any good for non-EndlessOS users, and EndlessOS users will have the extension installed automatically on update.

I guess it depends whether or not we want to enforce the icon theme setting in eos-knowledge-lib.

Reply:
At first guess I think we would not want to enforce the icon theme in eos-sdk apps on non-EndlessOS desktops.

Reply:
Fair enough

Reply:
Actually I think its better to let flatpak automatically install the extension since it wont affect non EOS users at all because they wont have EndlessOS as their icontheme and like @ebassi pointed out we do want it to be installed automatically for EOS users